src/Mvc/Mvc.Core/src/Builder/ControllerEndpointRouteBuilderExtensions.cs

```diff
@@ -106,7 +106,7 @@ public static IEndpointConventionBuilder MapDefaultControllerRoute(this IEndpoin
 
         /// <summary>
         /// Adds endpoints for controller actions to the <see cref="IEndpointRouteBuilder"/> and specifies a route
-        /// with the given <paramref name="name"/>, <paramref name="areaName"/>, <paramref name="template"/>, 
```

Comment:
VS is doing a format.

Reply:
For anyone who doesn't already know: adding `?w=1` to the end of the url hides whitespace only diff.